### PR TITLE
Clicking on the answers/buttons takes you to the proper landing pages

### PR DESCRIPTION
**Pull request by smarthutza**
Clicking on the answers/buttons takes you to the proper landing pages

### CODE REVIEW

File: public/js/logic.js [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+
+// document.querySelectorAll('figure.three-videos').forEach((e, i) => {
+//   e.addEventListener('click', () => {
+//     console.log(`I am answer ${i + 1}`);
+//   });
+// });
+
+document.querySelectorAll('figure.five-videos, figure.three-videos, figure.two-videos, button#confirm-survey, button#ok-submit-survey, button#pre-session-button, button#post-session-button').forEach((e, i) => {
+  e.addEventListener('click', changePage)
+})
```
Comment (des-des) [inline public/js/logic.js]: hmm okay can you put all your js files in this directory, as right now you have some in `public` and some in `public/js` which is inconsistent.

File: src/views/layout/default.hbs [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
   {{{content}}}
 
+  <script type='text/javascript' src='./js/logic.js'></script>
   <script src="./session-storage.js"></script>
```
Comment (des-des) [inline src/views/layout/default.hbs]: Opts missed this but it is missing `type="text/javascript"`

**Review by des-des — CHANGES_REQUESTED** [review]

File: public/js/logic.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 function changePage () {
```
Comment (des-des) [inline public/js/logic.js:1]: okay this file is now duplicate

**Review by des-des — CHANGES_REQUESTED** [review]

File: public/js/logic.js (hunk 3 above)
Comment (lucyrose93) [reply public/js/logic.js:1]: sorted

**Comment by lucyrose93** [comment]
please merge @des-des

**Review by des-des — APPROVED** [review]

**Comment by des-des** [comment]
@lucyrose93 okay can merge now, jus fix conflicts then re-assign

**Comment by lucyrose93** [comment]
thanks @des-des 
ready to merge

**Pull request merged**